pycharm_debugger.py

```diff
@@ -0,0 +1,12 @@
+def set_trace():
+    import pydevd_pycharm
+
+    gateway_ip = "172.17.0.1"
+    # Set up the debugger to connect to the host via the gateway IP
+    pydevd_pycharm.settrace(
+        host=gateway_ip,
+        port=12345,  # Use the same port number configured in PyCharm
+        stdoutToServer=True,
+        stderrToServer=True,
+        # suspend=True
```

Comment:
What's this option for?

Reply:
Good point, I've changed that now so it's set better. `suspend=True` means it stops at `breakpoint()`.

What's really annoying is that by default (suspend=True) the context in your IDE gets moved to this module which takes you away from the code you want to be looking at. 
Explicitly setting suspend=False (as I've done now) stops that, it now only stops at the breakpoints set in the gutter.

I've also added an image for the matching pycharm configuration to the docs.